pkg/logging/scalyr/create.go

```diff
@@ -0,0 +1,71 @@
+package scalyr
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/fastly"
+)
+
+// CreateCommand calls the Fastly API to create Scalyr logging endpoints.
+type CreateCommand struct {
+	common.Base
+	manifest manifest.Data
+	Input    fastly.CreateScalyrInput
+}
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent common.Registerer, globals *config.Data) *CreateCommand {
+	var c CreateCommand
+
+	// Without this, parsing flags leads to panics.
+	// This is because if we try to scan directly into c.Input.<Field> without first
+	// creating a valid reference, flags will attempt to be scanned into null pointers.
+	// This will result in sending the zero value values for unset flags which is acceptable
+	// for Creates, but not Updates.
+	c.Input = fastly.CreateScalyrInput{
+		Name:              fastly.String(""),
+		Format:            fastly.String(""),
+		FormatVersion:     fastly.Uint(0),
+		Token:             fastly.String(""),
+		ResponseCondition: fastly.String(""),
+		Placement:         fastly.String(""),
+	}
+
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("create", "Create a Scalyr logging endpoint on a Fastly service version").Alias("add")
+
+	c.CmdClause.Flag("name", "The name of the Scalyr logging object. Used as a primary key for API access").Short('n').Required().StringVar(c.Input.Name)
```

Comment:
```suggestion
	c.CmdClause.Flag("name", "The name of the Scalyr logging object. Used as a primary key for API access").Short('n').Required().StringVar(&c.Input.Name)
```
(suggestion just to highlight inconsistency, it is not a valid change)

Note the inconsistency with other "super" fields.